lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -424,8 +424,6 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         if vector.is_empty() {
             return Ok(vec![]);
         }
-        let mut vector = vector.clone();
```

Comment:
@timvisee here is the meaningful change

Reply:
Woah, that clearly shows 'sort' as well! The rest of the diff just distracted me.

Thanks!

Reply:
My bad, I did not push a nice diff on this one, sorry.